templates/footer.html

```diff
@@ -1,5 +0,0 @@
-<div class="docs-rs-footer">
```

Comment:
Strangely enough, this file is not marked as deleted. Github UI bug?

Reply:
Nope, not a GitHub UI bug - I had deleted the contents but not the file. In my most recent push it is now properly deleted.